set.mm

```diff
@@ -217908,6 +217908,46 @@ an extension of the previous (inserting an element and its inverse at
       WKWFWCWGABCDGHIVARAVKCVIEVNVOHWIWHJTQVB $.
   $}
 
+  ${
+    $d .+ x y  $.  $d .+^ a b x y $.  $d F a b x y $.  $d G x y $.  $d H i j $.
+    $d X a b x y $.  $d Y a b i j x y $.  $d a b i j x y ph $.
+    ghmabl.x $e |- X = ( Base ` G ) $.
+    ghmabl.y $e |- Y = ( Base ` H ) $.
+    ghmabl.p $e |- .+ = ( +g ` G ) $.
+    ghmabl.q $e |- .+^ = ( +g ` H ) $.
+    ghmabl.f $e |- ( ( ph /\ x e. X /\ y e. X )
+      -> ( F ` ( x .+ y ) ) = ( ( F ` x ) .+^ ( F ` y ) ) ) $.
+    ghmabl.1 $e |- ( ph -> F : X -onto-> Y ) $.
+    ${
+      ghmcmn.3 $e |- ( ph -> G e. CMnd ) $.
+      $( The image of a commutative monoid` G ` under a group homomorphism
+         ` F ` is a commutative monoid.  (Contributed by Thierry Arnoux,
+         26-Jan-2020.) $)
+      ghmcmn $p |- ( ph -> H e. CMnd ) $=
```

Comment:
Is there a version of this for monoids/groups?

Reply:
Also, it would be nice to conclude that `F` is also a group hom.

Reply:
Yes, [~mhmmnd](http://us2.metamath.org/mpeuni/mhmmnd.html) is the version for monoids, and [~ghmgrp](http://us2.metamath.org/mpeuni/ghmgrp.html) for groups.

Reply:
The proof that `F` is a group homomorphism is in the [last commit](https://github.com/metamath/set.mm/pull/1442/commits/a06956372243979818c3a30cef91945ec9cb2bb6).